__tests__/examples/webpack.skpm.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+
+const pkg = require(path.resolve(__dirname, 'package.json'));
+
+const aliasedModules = Object.keys(pkg.dependencies);
+
+module.exports = (config) => {
+  config.resolve = {
+    ...config.resolve,
+    alias: {
+      ...config.resolve.alias,
+      'react-sketchapp': path.resolve(__dirname, '../../lib'),
```

Comment:
Seems that `@skpm/test-runner` needs the commonjs build, and doesn't work with the ES build, strange.